docs/sphinx/source/whatsnew/v0.6.1.rst

```diff
@@ -73,6 +73,8 @@ Bug fixes
 * Fix documentation errors when using IPython >= 7.0.
 * Fix error in :func:`pvlib.modelchain.ModelChain.infer_spectral_model` (:issue:`619`)
 * Fix error in ``pvlib.spa`` when using Python 3.7 on some platforms.
+* Fix error in :func:`pvlib.irradiance._delta_kt_prime_dirint` (:issue: `637`). The error affects
```

Comment:
I think the space between `:issue:` and the backtick will cause a problem for the automated issue linker.